fedbiomed/common/secagg_manager.py

```diff
@@ -113,30 +122,60 @@ def _add_generic(self, secagg_id: str, parties: List[str], specific: dict):
             FedbiomedSecaggError: failed to insert in database
             FedbiomedSecaggError: an entry already exists for `secagg_id` in the table
         """
-        if self._get_generic(secagg_id) is not None:
-            errmess = f'{ErrorNumbers.FB623.value}: error adding element in table "{self._table}": ' \
-                      f' an entry already exists for secagg_id={secagg_id}'
-            logger.error(errmess)
-            raise FedbiomedSecaggError(errmess)
-
+        self._check_existing_entry_in_db(secagg_id)
         specific.update({
             'secagg_version': str(__secagg_element_version__),
             'secagg_id': secagg_id,
-            'parties': parties
+            'parties': parties,
+            'secagg_scheme': secagg_scheme.value,
```

Comment:
Hi @ybouilla 

I don't understand why the `secagg_scheme` is needed ? (moreover we save it but don't really use it)
Isn't the `secagg_elem` enough ?

Reply:
Actually it was in provision for future developments. I thought it could be useful to treat some context regarding its scheme. But if you think it is meaningless we can remove it.

Reply:
I think it is not needed.

But it's not harmful either ...

Reply:
... but if we keep it, then we should check it also every time we check the `secagg_elem` ... so it's probablly better to remove them to avoid useless code checks.